hivemind/optim/grad_scaler.py

```diff
@@ -33,8 +32,8 @@ def running_global_step(self):
         finally:
             self._is_running_global_step = was_running
 
-    def unscale_(self, optimizer: Optimizer) -> bool:
-        assert isinstance(optimizer, DecentralizedOptimizerBase)
+    def unscale_(self, optimizer: TorchOptimizer) -> bool:
+        assert hasattr(optimizer, "opt"), "hivemind.GradScaler only supports hivemind optimizer wrappers"
```

Comment:
What if there's a non-hivemind wrapper of TorchOptimizer? A more explicit check would use `isinstance` IMO

Reply:
[done]